.codespell-whitelist.txt

```diff
@@ -0,0 +1,2 @@
+nd
+cace
```

Comment:
What was "cace" for?

Reply:
CACE was for below but codespell is case insensitive for the file version of ignore. But this will get clarified when I migrate to having this be a dictionary plus an ignore file

Reply:
Thanks for the clarification